examples/react/react-app/dojoConfig.ts

```diff
@@ -3,4 +3,6 @@ import { createDojoConfig } from "@dojoengine/core";
 
 export const dojoConfig = createDojoConfig({
     manifest,
+    relayUrl:
+        "/ip4/127.0.0.1/udp/9091/webrtc-direct/certhash/uEiCAoeHQh49fCHDolECesXO0CPR7fpz0sv0PWVaIahzT4g",
```

Comment:
can we standardise this ?

Reply:
oh i should've gitignored that. the relay url for wasm is dynamic so we can't standarize it